### DISCUSSION TITLE
add description meta

### DISCUSSION THREAD

**Discussion by Zebeqo** [jotaijs/jotai-devtools#95] [Ideas]
If we can add the description of the atom through `countAtom.debugDescription = 'number of player failures'` and display it in the Meta of jotai-devtools, it will be very helpful for non-developers to understand the state of the app.

**Comment by dai-shi** [comment]
I wonder if TS module augmentation would work.

**Comment by arjunvegda** [comment]
That's an interesting idea @Zebeqo! I'd love to understand the full use case. Do the non-developers in your team run your app locally? 🤔 

I wonder if having `debugDescription` would be difficult to maintain on client apps in the long run. Any thoughts on having a concise `debugLabel` instead?  For eg. `numOfPlayerFailuresAtom`

**Reply by Zebeqo** [reply]
After our discussion it turns out that you are right, a concise `debugLabel` should be enough for our team to understand most atoms. Thank you for your answer!
